### PR TITLE
add fields for fines block

### PR DESCRIPTION
**Pull request by EddieOrbit**
### Added

- Field with path `fines.items[].fssp.date`
- Field with path `fines.items[].fssp.is_proceed`

## Checklist

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I wrote unit tests for my code _(if tests is required for my changes)_
- [X] I have made changes in `CHANGELOG.md` file

### CODE REVIEW

File: reports/default/json-schema.json [hunk 1]
```diff
@@ -4721,6 +4721,42 @@
...
+                                        "oneOf": [
+                                            {
+                                                "type": "null"
+                                            },
+                                            {
+                                                "$ref": "#/definitions/datetime"
+                                            }
+                                        ],
+                                        "examples": [
+                                            "2018-05-23"
```
Comment (Zaboday) [inline reports/default/json-schema.json:4739]: Указан объект `datetime` а пример приведен как `date`

File: reports/default/examples/full.json [hunk 2]
```diff
@@ -793,6 +793,10 @@
                 "location": {
                     "raw": "МОСКВА Г. ПАРКОВКА №4034 (50) ПРОФСОЮЗНАЯ УЛИЦА, ДОМ 12"
                 },
+                "fssp": {
+                    "date": "2018-05-23 00:00:00",
```
Comment (Zaboday) [inline reports/default/examples/full.json:797]: А в процессинге отдаст `date`. Где Правда?

Comment (EddieOrbit) [reply reports/default/examples/full.json:797]: Получается здесь

File: reports/default/json-schema.json (hunk 1 above)
Comment (EddieOrbit) [reply reports/default/json-schema.json]: Исправил

**Pull request merged**